src/Forms/ConfirmedPasswordField.php

```diff
@@ -136,6 +136,9 @@ public function __construct(
         $titleConfirmField = null
     ) {
 
+        // This param is requested only for backward compat
+        $form;
```

Comment:
Maybe `ignoreErrors:` is a better place than this noop.

Reply:
I like this better.